usaspending_api/agencies/README.md

```diff
@@ -0,0 +1,77 @@
+# A Little Background
+
+Whenever an award or subaward is made, there are several federal agencies tied to the award
+including, but not limited to:
+
+* Awarding Agency - The agency that interacted with the recipient to make the award.
+* Funding Agency - The agency that provided the bulk of the funding for the award.
+* Reporting Agency - The agency that reported the award to the system of record (FSRS, DABS, etc).
+
+These agencies may overlap or be missing entirely from the system of record.
```

Comment:
`or be missing entirely from the system of record.` suggests that they can point to an agency which doesn't exist, as opposed to just being null